types/ldap-escape/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for ldap-escape 2.0
+// Project: https://github.com/tcort/ldap-escape
+// Definitions by: Simone Corsi <https://github.com/simonecorsi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function filter(TemplateStringsArray: TemplateStringsArray, ...str: string[] | number[]): string;
+export function dn(TemplateStringsArray: TemplateStringsArray, ...str: string[] | number[]): string;
+
+export as namespace LdapEscape;
```

Comment:
Why do we need this `export as namespace`? Looking at the original code, it seems to me the original library only has named exports, so I think the right thing to do is to not have the `export as namespace`, unless I'm missing something.

Reply:
You're right, that was a biased change I didn't think enough before doing, I'm gonna change it!